.github/workflows/reusable-docker-arm-build.yaml

```diff
@@ -142,20 +139,16 @@ jobs:
         if: ${{ github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v')) }}
         uses: docker/build-push-action@v6
         with:
-          context: images/
-          file: images/Dockerfile.kairos-${{ inputs.family }}
+          context: .
+          file: images/Dockerfile
           platforms: linux/arm64
-          push: "true"
-          tags: ${{ env.IMAGE_REF }}
+          push: true
+          tags: ${{ env.IMAGE_REF }}, kairos:${{ env.GIT_VERSION }}
```

Comment:
the second tag here will be pushed multiple times no? I mean from every job that uses this same pipeline on the same commit, no matter what flavor it's building. Do we want that?

Reply:
yep yep, I dropped already, good catch!